docs/bigquery-usage.rst

```diff
@@ -232,11 +232,7 @@ Get rows from a table's data:
    >>> client = bigquery.Client()
    >>> dataset = client.dataset('dataset_name')
    >>> table = dataset.table(name='person_ages')
-   >>> rows, next_page_token = table.data(max_results=100)  # API request
-   >>> rows.csv.headers
```

Comment:
How can a user access the field names from the schema? Does it come down with `fetch_data`?